docs/_posts/examples/3400-01-24-drag-a-point.html

```diff
@@ -0,0 +1,166 @@
+---
+layout: example
+category: example
+title: "Create a draggable point"
+description: "Drag the point to a new location on a map and populates its coordinates in a display."
+---
+
+<style>
+.coordinates {
```

Comment:
Using an id to style something in CSS is considered bad practice. It applies too much specificity and we shouldn't employ that practice.